src/main/java/com/targomo/client/api/TravelOptions.java

```diff
@@ -946,6 +965,8 @@ public String toString() {
         builder.append(multiGraphAggregationMaxResultValue);
         builder.append("\n\tmultiGraphAggregationMathExpression: ");
         builder.append(multiGraphAggregationMathExpression);
+        builder.append("\n\tmultigraphLayerCustomGeometryMergeAggregation: ");
+        builder.append(multigraphLayerCustomGeometryMergeAggregation);
```

Comment:
one tiny thing. I've noticed we usually write `multigraph` in all the other parameters with a capital `G`, i.e. this parameter should be renamed to `multiGraphLayerCustomGeometryMergeAggregation`

sorry I was giving you the wrong name earlier